recipes/cyclus/build.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+set -e
+
+source activate "${CONDA_DEFAULT_ENV}"
+
+UNAME="$(uname)"
+#export CFLAGS="${CFLAGS} -O3"
+#export CXXFLAGS="${CXXFLAGS} -O3"
+#export LIBRARY_PATH="${PREFIX}/lib:${LIBRARY_PATH}"
+#export INCLUDE_PATH="${PREFIX}/include:${INCLUDE_PATH}"
+
+#export PATH="${PREFIX}/bin:${PATH}"
+#export LD_LIBRARY_PATH="${PREFIX}/lib:${LD_LIBRARY_PATH}"
+#export CMAKE_LIBRARY_PATH="${PREFIX}/lib:${CMAKE_LIBRARY_PATH}"
+
+if [ "${UNAME}" == "Darwin" ]; then
+  # for Mac OSX
+  #export CC=gcc
+  #export CXX=g++
+  #export CC=clang
+  #export CXX=clang++
+  export MACOSX_VERSION_MIN="10.7"
+  #export CXXFLAGS="${CXXFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+  #export CXXFLAGS="${CXXFLAGS} -stdlib=libc++"
+  #export CXXFLAGS="${CXXFLAGS} -std=c++11"
+  #export LDFLAGS="${LDFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+  #export LDFLAGS="${LDFLAGS} -stdlib=libc++ -lc++"
+  #export LDFLAGS="${LDFLAGS} -std=c++11 -lc++"
+  #export LINKFLAGS="${LDFLAGS}"
+  export CMAKE_OSX_DEPLOYMENT_TARGET="${MACOSX_VERSION_MIN}"
```

Comment:
Adding in this PR ( https://github.com/conda-forge/toolchain-feedstock/pull/5 ).